docs/Auth-with-Ionic3-Angular4.md

```diff
@@ -0,0 +1,692 @@
+# Using AngularFire2 with Ionic 3-Angular 4
+
+This tutorial provides a walkthrough of integrating AngularFire2 Authentication with Ionic3/Angular4. 
+The below setup has been tested on Windows 10, but it should be same for Mac/Linux.
+
+Note: - If you're working with Ionic2 and Angular2.0, then you should refer to **Auth-with-Ionic2** tutorial 
+[here](https://github.com/angular/angularfire2/blob/master/docs/Auth-with-Ionic2.md)
+
+Ensure that you're executing these commands as **Administrator** on Windows and **sudo** on Mac/Linux to avoid any errors.
+
+This tutorial uses **Facebook** as the sign-in provider. After completion of this tutorial, you should be able to configure
+other sign-in providers like **Twitter**, **Google** on your own.
+
+### Prerequisites
+The first step is to ensure you've latest version of **Node** installed.
+You can get the latest version from [here](https://nodejs.org).
+This will install both node and npm.
+
+After installing node, check the version by executing the following command in your prompt window.
+
+```bash
+
+C:\projects>node -v
+v6.10.2
+
+```
+As of writting this document, this is the most stable version. If you're not on this version,
+please upgrade yourself to latest version by installing node from [here](https://nodejs.org).
+
+Check your npm version by executing the following command.
+```bash
+
+C:\projects>npm -v
+3.10.10
+
+```
+Install the latest verion of `angular4` aka `angular`, by executing the command
```

Comment:
Hi @katowulf , accepted your suggestion and updated the document.
I can confirm, the application works on both desktop browser and mobile phone.